### PR TITLE
Feature/aqm canopy new fix

### PR DESCRIPTION
**Pull request by iri01**
## Description

(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)
Provide a detailed description of what this PR does.  
What bug does it fix, or what feature does it add?  
Is a change of answers expected from this PR?  

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues should always be created before starting work on a PR branch!)
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested?  
What compilers / HPCs was it tested with?  
Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)  
Have the ufs-weather-model regression test been run? On what platform?  
- Will the code updates change regression test baseline? If yes, why? Please show the baseline directory below.
- Please commit the regression test log files in your ufs-weather-model branch

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
Those branches should have matching names (ideally)

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

File: ccpp/physics
Comment (drnimbusrain) [inline ccpp/physics:1]: @iri01 Let's discuss this submodule update.  Are there any new changes to ccpp-physics associated with this fv3atm PR?  I had already updated ccpp-physics submodule here and likely you should not need be updating the submodule here.  Remember we only want to update what we have changed.

File: ccpp/data/GFS_typedefs.F90
Comment (drnimbusrain) [inline ccpp/data/GFS_typedefs.F90:1]: @iri01 I don't see any changes here that would change  how the canopy variables are assigned.

File: ccpp/data/GFS_typedefs.meta
Comment (drnimbusrain) [inline ccpp/data/GFS_typedefs.meta:1]: @iri01 I do not see what has changed here in this PR. It seems like this is just a delete and recopy.

File: ccpp/data/GFS_typedefs.meta [hunk 1]
```diff
@@ -8274,6 +8237,43 @@
...
+  standard_name = canopy_clumping_index
+  long_name = canopy clumping index
+  units = none
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+[cpopu]
+  standard_name = canopy_population_density
+  long_name = population density used for canopy correction
+  units = none
```
Comment (drnimbusrain) [inline ccpp/data/GFS_typedefs.meta:8272]: Also, [I had made a previous change to this file for cpopu units,](https://github.com/noaa-oar-arl/fv3atm/commit/56080497d7f3afca87e4a86c2b9a88e4f787d57a) but I don't see it here in your branch.  Please be sure to update to latest before making any changes on top.  Likely we need to discuss again.

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@iri01 Let's discuss.

File: ccpp/physics
Comment (drnimbusrain) [reply ccpp/physics:1]: Also, including the ccpp-physics submodule update here then has conflicts as you can see below.

File: ccpp/data/GFS_typedefs.F90
Comment (iri01) [reply ccpp/data/GFS_typedefs.F90:1]: Just added comments for clarity. The change is in the meta file.

File: ccpp/data/GFS_typedefs.meta
Comment (iri01) [reply ccpp/data/GFS_typedefs.meta:1]: The list of the canopy vars is now under category `GFS_diag_type` , while formerly it was under category `GFS_sfcprop_type`. The definitions are the same. But you are right, I've lost your modifications to the units on FCH and POPU. 

[ccpp-table-properties]
  name = GFS_diag_type
  type = ddt
  dependencies =

[ccpp-arg-table]
  name = GFS_diag_type
  type = ddt
  ...

[claie]
  standard_name = canopy_leaf_area_index
  long_name = canopy leaf area index
  units = none
  dimensions = (horizontal_loop_extent)
  type = real
  kind = kind_phys

Comment (drnimbusrain) [reply ccpp/data/GFS_typedefs.meta:1]: OK, should have noticed the line number changes to be farther down.  Yes, make sure you have my latest changes to both cfch and cpopu units here in  meta.

**Pull request closed without merging**